homeassistant/config_entries.py

```diff
@@ -2894,14 +2894,20 @@ def async_get_supported_subentry_types(
 
     @callback
     def _async_abort_entries_match(
-        self, match_dict: dict[str, Any] | None = None
+        self,
+        match_dict: dict[str, Any] | None = None,
     ) -> None:
         """Abort if current entries match all data.
 
         Requires `already_configured` in strings.json in user visible flows.
```

Comment:
Please update the docstring to explain we won't abort a reconfigure or reauth flow when matching on the entry being reconfigured

Reply:
Updated the docstring